backend/pixeldrain/pixeldrain.go

```diff
@@ -0,0 +1,605 @@
+// Package pixeldrain provides an interface to the Pixeldrain object storage
+// system.
+package pixeldrain
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"path"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/rclone/rclone/fs"
+	"github.com/rclone/rclone/fs/config/configmap"
+	"github.com/rclone/rclone/fs/config/configstruct"
+	"github.com/rclone/rclone/fs/fshttp"
+	"github.com/rclone/rclone/fs/hash"
+	"github.com/rclone/rclone/lib/rest"
+)
+
+const timeFormat = time.RFC3339Nano
+
+// Register with Fs
+func init() {
+	fs.Register(&fs.RegInfo{
+		Name:        "pixeldrain",
+		Description: "Pixeldrain Filesystem",
+		NewFs:       NewFs,
+		Config:      nil,
+		Options: []fs.Option{{
+			Name: "api_key",
+			Help: "API key for your pixeldrain account.\n" +
+				"Found on https://pixeldrain.com/user/api_keys.",
+			Sensitive: true,
+		}, {
+			Name: "directory_id",
+			Help: "Root of the filesystem to use. Set to 'me' to use your personal filesystem.\n" +
```

Comment:
Note that the first line of these `Help:` lines is used as the help for the command line flag.

I think these are easier to read on multiple lines, so I think this should be

```go
Help: `Root of the filesystem to use.

Set to "me" to use your personal filesystem. Set to a shared directory ID to use a shared directory.`,
```